src/Thesis/QueryRunner.hs

```diff
@@ -0,0 +1,15 @@
+module Thesis.QueryRunner where
+
+import Thesis.Microtransaction (runMicrotransaction)
+import Thesis.LaplaceNoise (generate)
+import Thesis.Query
+import Thesis.SqlRunner (runSql)
+import System.Random (getStdGen)
+
+runQuery :: Query -> IO Text
+runQuery query = do
+  let queryResult = runMicrotransaction timeout (defaultAnswer query) (runSql query)
+  let gen = getStdGen
```

Comment:
I believe that it would lead to cleaner to design to not take stdGen from the IO monad, but either
1) pass it in an argument
2) take it from state from a custom state monad

To allow users to provide whatever stdGen they want.